app/assets/javascripts/proxy_status/dhcp.js

```diff
@@ -0,0 +1,63 @@
+function activateDhcpTables() {
```

Comment:
this is bad design, I'd go for improving it instead of further digging a hole.
also, if can't be avoided, use one function on the global namespace, but I really prefer changing the way this is handled.